config/esp32/components/chip/CMakeLists.txt

```diff
@@ -226,6 +226,7 @@ target_include_directories(${COMPONENT_LIB} INTERFACE
     "${CMAKE_CURRENT_BINARY_DIR}/src/include"
     "${CMAKE_CURRENT_BINARY_DIR}/include"
     "${CMAKE_CURRENT_BINARY_DIR}/gen/include"
+    "${CONFIG_CHIP_EXTERNAL_PLATFORM_INCLUDE}"
```

Comment:
Since the platform target is the relative path to this folder, how about using the relative path for platform_include as well?